index.html

```diff
@@ -64,12 +64,12 @@ <h2>work</h2>
     </section>
     <section class="about">
       <h2>about</h2>
-      <section class="about-content">
-        <section class="about-paragraph">
-          <p>
+      <section class="content">
+        <section class="paragraph">
+          <div>
             Rufio has drawn henna designs for years. In 2020, she did Inktober and began applying henna patterns to prompts. She has been drawing them ever since.
-          </p>
-          <p>Now, you can redeem one of her henna postcards in stream. </p>
+          </div>
+          <div>Now, you can redeem one of her henna postcards in stream. </div>
```

Comment:
Changed the `p` tags to `div` tags because for some reason the `p` tag forces a `margin-right` to the styling which was messing with the `img` sizing.